### PR TITLE
[v8.0.x] ReleaseNotes: Updated changelog and release notes for 8.0.4

### PR DESCRIPTION
**Pull request by grafanabot**
Backport 3cce67c044b2516d324959a80857cb9180dd1b5e from #36347

### CODE REVIEW

**Review by hugohaggmark — APPROVED** [review]
LGTM!

File: docs/sources/release-notes/release-notes-8-0-4.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+* **PieChart:** Sort legend descending, update placeholder to show default …. [#36062](https://github.com/grafana/grafana/pull/36062), [@ashharrison90](https://github.com/ashharrison90)
+* **TimeSeries panel:** Do not reinitialize plot when thresholds mode change. [#35952](https://github.com/grafana/grafana/pull/35952), [@dprokop](https://github.com/dprokop)
+
+### Bug fixes
+
+* **Elasticsearch:** Allow case sensitive custom options in date_histogram interval. [#36168](https://github.com/grafana/grafana/pull/36168), [@Elfo404](https://github.com/Elfo404)
+* **Elasticsearch:** Restore previous field naming strategy when using variables. [#35624](https://github.com/grafana/grafana/pull/35624), [@Elfo404](https://github.com/Elfo404)
+* **Explore:** Fix import of queries between SQL data sources. [#36210](https://github.com/grafana/grafana/pull/36210), [@ivanahuckova](https://github.com/ivanahuckova)
+* **InfluxDB:** InfluxQL query editor: fix retention policy handling. [#36022](https://github.com/grafana/grafana/pull/36022), [@gabor](https://github.com/gabor)
```
Comment (achatterjee-grafana) [inline docs/sources/release-notes/release-notes-8-0-4.md:22]:
```suggestion
* **InfluxDB:**  Fix retention policy handling in InfluxQL query editor. [#36022](https://github.com/grafana/grafana/pull/36022), [@gabor](https://github.com/gabor)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,25 @@
...
+* **PieChart:** Sort legend descending, update placeholder to show default …. [#36062](https://github.com/grafana/grafana/pull/36062), [@ashharrison90](https://github.com/ashharrison90)
+* **TimeSeries panel:** Do not reinitialize plot when thresholds mode change. [#35952](https://github.com/grafana/grafana/pull/35952), [@dprokop](https://github.com/dprokop)
+
+### Bug fixes
+
+* **Elasticsearch:** Allow case sensitive custom options in date_histogram interval. [#36168](https://github.com/grafana/grafana/pull/36168), [@Elfo404](https://github.com/Elfo404)
+* **Elasticsearch:** Restore previous field naming strategy when using variables. [#35624](https://github.com/grafana/grafana/pull/35624), [@Elfo404](https://github.com/Elfo404)
+* **Explore:** Fix import of queries between SQL data sources. [#36210](https://github.com/grafana/grafana/pull/36210), [@ivanahuckova](https://github.com/ivanahuckova)
+* **InfluxDB:** InfluxQL query editor: fix retention policy handling. [#36022](https://github.com/grafana/grafana/pull/36022), [@gabor](https://github.com/gabor)
```
Comment (achatterjee-grafana) [inline CHANGELOG.md:17]:
```suggestion
* **InfluxDB:** Fix retention policy handling in InfluxQL query editor. [#36022](https://github.com/grafana/grafana/pull/36022), [@gabor](https://github.com/gabor)
```

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
Minor doc change req.

**Review by achatterjee-grafana — APPROVED** [review]

**Pull request merged**